### DISCUSSION TITLE
Group Leader - Seeking

### DISCUSSION THREAD

**Discussion by FrozenEmpire** [ChilliHugger/The-Lords-Of-Midnight#189] [Code Algorithm Questions] [answered]
Hi Chris,

If a group leader performs a SEEK (either via command in LOM, or auto-seek in DDR), and finds a THING which would adjust the time of day, or energy levels etc, then I believe currently it only affects the group leader, and not his followers, right ?

So if for example the Flames of Dawn are found, the group leader's time resets to dawn, and the followers do not, and they will then likely hold up the group from moving at some point soon.

Was that the intended behaviour ?

Thanks

**Comment by IcemarkUK** [comment, marked as answer]
Yes this was the intended behaviour. I decided that the affects were localised and didn't have a blast radius. I was also conscious that the player could use the grouping to 'game' the system. ie: group people, seek and find a cup of dreams, everyone gets it, then ungroup. (Albeit that this would be more LoM where they are predictable.)

There is an argument that maybe a couple could be given to the whole group - Shelter, Wild horses..

**Reply by FrozenEmpire** [reply]
Thanks, yeah I see your point. It could be used to get a competitive advantage compared to vanilla DDR

But on the other hand, being grouped kind of limits the benefit of eg the flames of dawn, if the followers will drag the group down soon after.

Maybe that’s the price to pay for the benefits of moving as a group :)
